static/_redirects

```diff
@@ -0,0 +1,222 @@
+/docs/references/openid-connect /end-user-flows/sign-up-and-sign-in 301
+/docs/references/openid-connect/ /end-user-flows/sign-up-and-sign-in 301
+/docs/references/openid-connect/authentication-parameters /end-user-flows/authentication-parameters 301
+/docs/references/openid-connect/authentication-parameters/ /end-user-flows/authentication-parameters 301
+/docs/references/openid-connect/backchannel-logout /end-user-flows/sign-out#federated-sign-out-back-channel-logout 301
+/docs/references/openid-connect/backchannel-logout/ /end-user-flows/sign-out#federated-sign-out-back-channel-logout 301
+/docs/references/openid-connect/introspect-tokens /concepts/opaque-token
+/docs/references/openid-connect/introspect-tokens/ /concepts/opaque-token
+/docs/references/openid-connect/signing-keys-rotation /developers/signing-keys 301
+/docs/references/openid-connect/signing-keys-rotation/ /developers/signing-keys 301
+/docs/references/resources /authorization/api-resources 301
+/docs/references/resources/ /authorization/api-resources 301
+/docs/references/sdk-convention /developers/sdk-conventions 301
+/docs/references/sdk-convention/ /developers/sdk-conventions 301
+/docs/references/sdk-convention/core-sdk-convention /developers/sdk-conventions/core-sdk-convention 301
+/docs/references/sdk-convention/core-sdk-convention/ /developers/sdk-conventions/core-sdk-convention 301
+/docs/references/sdk-convention/design-strategy /developers/sdk-conventions/design-strategy 301
+/docs/references/sdk-convention/design-strategy/ /developers/sdk-conventions/design-strategy 301
+/docs/references/sdk-convention/platform-sdk-convention /developers/sdk-conventions/platform-sdk-convention 301
+/docs/references/sdk-convention/platform-sdk-convention/ /developers/sdk-conventions/platform-sdk-convention 301
+/docs/references/tenants /logto-cloud/tenant-settings#production 301
+/docs/references/tenants/ /logto-cloud/tenant-settings#production 301
+/docs/references/translate-cli /logto-oss/translate-cli 301
+/docs/references/translate-cli/ /logto-oss/translate-cli 301
+/docs/references/tunnel-cli /customization/bring-your-ui/debug-and-test-your-custom-ui-locally 301
+/docs/references/tunnel-cli/ /customization/bring-your-ui/debug-and-test-your-custom-ui-locally 301
+/docs/references/tunnel-cli/deploy /customization/bring-your-ui/upload-custom-ui-assets-using-cli 301
+/docs/references/tunnel-cli/deploy/ /customization/bring-your-ui/upload-custom-ui-assets-using-cli 301
+/docs/references/users /user-management/user-data 301
+/docs/references/users/ /user-management/user-data 301
+/docs/references/users/custom-data /user-management/user-data#custom-data 301
+/docs/references/users/custom-data/ /user-management/user-data#custom-data 301
+/docs/references/users/social-identities /user-management/user-data#social-identities 301
+/docs/references/users/social-identities/ /user-management/user-data#social-identities 301
+/docs/references/using-cli/* /logto-oss/using-cli/:splat 301
```

Comment:
Just curious what is `:splat`?

Reply:
Aha, get it.

> On matching, a splat (asterisk, *) will greedily match all characters. You may only include a single splat in the URL.

Reply:
https://developers.cloudflare.com/pages/configuration/redirects/#splats

Reply:
yep, found it